website/content/en/docs/advanced-topics/heterogeneous.md

```diff
@@ -0,0 +1,400 @@
+
+#### Integrating with Operator Lifecycle Manager
+
+The Operator Lifecycle Manager (OLM) is often used to distribute operators via operator catalogs. The operator deployment object in an OLM integrated operator is defined in the `ClusterServiceVersion` yaml. It is important to remember to set the node affinity block in each of the `spec.template.spec` under `spec.install.deployments`.
+
+## Examples
+This section attempts to show how each of the sample operators could be updated to declare a set of supported architectures. Most of the time, these changes will be in `yaml` files, but different kinds of operators embed `PodSpec` and `PodTemplateSpec` objects in different places.
+
+The most common directory with objects that will need an architecture affinity block is the operator's `config` directory. Additionally, if the operator is configured for OLM, there will likely be a deployment object in the `bundle/manifests/*.clusterserviceversion.yaml`. These locations are common across all operator types.
+
+Each of these examples can be in the [OperatorSDK GitHub repository](https://github.com/operator-framework/operator-sdk).
+
+### Ansible Operators
+In the `testdata/ansible` samples app, you'll find several configuration objects that need a nodeAffinity specification in the `config` and `bundle/manifests` directories. The full list is as follows:
+- `testdata/ansible/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml`
+- `testdata/ansible/memcached-operator/config/default/manager_config_patch.yaml`
+- `testdata/ansible/memcached-operator/config/default/manager_auth_proxy_patch.yaml`
+- `testdata/ansible/memcached-operator/config/manager/manager.yaml`
+
+The Ansible example also has Pod-defining objects as part of the test directories. For completeness, these should also need to define a nodeAffinity specification. The operator author will know best if tests are expected to be run in mixed-architecture environments, but updating these helps future-proof the operator.
+- `testdata/ansible/memcached-operator/config/testing/debug_logs_patch.yaml`
+- `testdata/ansible/memcached-operator/config/testing/manager_image.yaml`
+- `testdata/ansible/memcached-operator/config/testing/pull_policy/Always.yaml`
+- `testdata/ansible/memcached-operator/config/testing/pull_policy/IfNotPresent.yaml`
+- `testdata/ansible/memcached-operator/config/testing/pull_policy/Never.yaml`
```

Comment:
The only Deployment define here is in the manager_image (but that is for test the project only)
Note that the other files are patches to add envars and etcs. 

Ansible/Helm also uses the same structure for Golang and the projects by default scaffolds one 2 workloads which are the containers for the manager and the proxy. 

Check the PR for the docs in KB: https://github.com/kubernetes-sigs/kubebuilder/pull/2906/files#diff-b79471388fbaeb59d4a9793846e4b1c0e533f34c4768b5fad4907338b90c74a8

Btw, I think we can link the KB doc and just add here what is SDK specific. 

c/c @jmrodri